doxia-integration-tools/src/main/resources/default-site.xml

```diff
@@ -22,6 +22,11 @@ under the License.
   <bannerLeft>
     <name>${project.name}</name>
   </bannerLeft>
+  <skin>
```

Comment:
so fluido will be default now?
the PR title was clear. I was thinking it was turning to "no more default skin now you have to configure one!"

Reply:
One of the goals is to evaluate whether the bundled skin can be removed to remove duplication between default and Fluido. Moreover, it will only be used if a site.xml has not been provided. Otherwise you need to provide one. This PR is incomplete because a few tests need to be fixed. Default and Fluido share 60% of the code, consider the maintenance work.

Reply:
The "no more default skin now you have to configure one!" has already been done in https://issues.apache.org/jira/browse/DOXIASITETOOLS-257